tests/Resources/Page/HybridCloudConsole/HCCLogin.robot

```diff
@@ -24,7 +24,7 @@ Login to HCC
     Input Text  id=password  ${password}
     Click Button    Log in
   END
-  Run Keyword And Continue On Failure    Wait For HCC Splash Page
+  Run Keyword And Ignore Error     Wait For HCC Splash Page
```

Comment:
Just a suggestion, Do you think it will be good idea to use "**Run Keyword And Warn On Failure**" . It will not fail the overall TC but will give warning so that we can get to know ?

Reply:
@tarukumar I think it's not worth because we are testing the splash page "feature", so we are not really interested in if it appears or not, the important thing is that the next keywords run successfully